.github/workflows/generate-changelogs.yml

```diff
@@ -0,0 +1,45 @@
+name: Generate Changelogs
+run-name: 'chore(release): generate changelogs for ${{ inputs.previous_release_tag }}..${{ inputs.future_release_tag }}'
+
+on:
+  workflow_dispatch:
+    inputs:
+      previous_release_tag:
+        required: true
+        description: previous release tag
+      future_release_tag:
+        required: true
+        description: future release tag
+
+jobs:
+  build:
+    name: Generate changelogs
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Generate release changelogs
+        uses: daeuniverse/changelogs-generator-action@test
```

Comment:
I've noticed we are picking the test branch of this action instead of default branch or a release tag, is this expected?

Reply:
Nope, will fix it in the last commit.